src/Event.php

```diff
@@ -33,8 +34,23 @@ public function acknowledge(Message $message)
         array_map(function ($name) use ($message) {
             $this->emitter->emit($name, $message);
         }, [
-            static::QUEUE_ACKNOWLEDGE,
-            sprintf('%s.%s', static::QUEUE_ACKNOWLEDGE, $message->handler())
+            static::MESSAGE_ACKNOWLEDGE,
+            sprintf('%s.%s', static::MESSAGE_ACKNOWLEDGE, $message->handler())
```

Comment:
@shadowhand I had thought about doing that, but I'm hoping I can get https://github.com/thephpleague/event/pull/73 merged in which will clean it up nicely.